.github/workflows/ubuntu.yml

```diff
@@ -166,33 +168,33 @@ jobs:
       - name: configure
         run: |
           cd _build
-          ../configure --enable-code-coverage CPPFLAGS="-Werror=declaration-after-statement"
+          ../configure --enable-code-coverage \
```

Comment:
it may be reasonable to also add `-pedantic` to CPPFLAGS and specify `CC="gcc -std=c89"` here